deployment/network-operator/charts/sriov-network-operator/templates/sriovnetworknodepolicy.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
can you submit one for sriov-network-operator ?

Reply:
here is it: https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/401